packages/polymath-issuer/src/pages/compliance/CompliancePage.js

```diff
@@ -719,40 +719,42 @@ class CompliancePage extends Component<Props, State> {
                     </div>
                   </div>
                 </Grid.Col>
-                <Grid.Col gridSpan={[12, 12, 6, 6]}>
-                  <div id="compliance">
-                    <br />
-                    <div className="pui-page-box compliance-form">
-                      <h1 className="pui-h1">No Partial Transfers</h1>
-                      <div className="whitelist-settings">
-                        <div className="bx--form-item">
-                          <label
-                            htmlFor="partialTransferToggle"
-                            className="bx--label"
+                {token.contract.version === '3.0.0' && (
```

Comment:
Since we need this feature enabled for `3.1.0` too, I think we could use `semvar` to make sure that `version` is `>= 3.0.0`.

Reply:
Done